src/main/java/org/jboss/xavier/integrations/migrationanalytics/business/VMWorkloadInventoryCalculator.java

```diff
@@ -75,6 +75,15 @@ private VMWorkloadInventoryModel createVMWorkloadInventoryModel(Map vmStructMap)
             model.setHasRdmDisk(hasRdmDisk);
         }
 
+        String cpuAffinity = readValueFromExpandedEnvVarPath(CPUAFFINITYPATH, vmStructMap);
+        if (cpuAffinity != null) {
```

Comment:
what do you think about reducing the code to 
`model.setCpuAffinityNotNull(cpuAffinity == true)`

Reply:
@jonathanvila , the business requirement is to raise a flag in xavier-analytics if cpuAffinity is null. So on the first point, I think that's a good idea, It's done.

Reply:
And I've changed the code to treat "" as null